assets/bem/styles.scss

```diff
@@ -14,6 +14,49 @@
 @import "blocks/list";
 @import "blocks/list-header";
 @import "blocks/book";
+@import "blocks/overlay"; 
+@import "blocks/pop-up";
+
+
```

Comment:
don't need that many empty lines, 1 should be enough

Reply:
Done!